tests/fixtures/module-unification-addon/.eslintrc.js

```diff
@@ -20,7 +20,6 @@ module.exports = {
     // node files
     {
       files: [
-        '.eslintrc.js',
```

Comment:
why did this change?

Reply:
and if we also have ` tests/fixtures/module-unification-addon/npm/.eslintrc.js` then what is this file here (` tests/fixtures/module-unification-addon/.eslintrc.js`) being used for?

Reply:
@Turbo87 

> why did this change?

The existing addon blueprint does not have this line. 

Do you want me to add it to both places ( blueprint file and the fixture)?


---------------------------------------------------------

`tests/fixtures/module-unification-addon/.eslintrc.js` is needed because how `checkEslintConfig` works, this function checks the `estlintrc.js` on the parent folder.


`tests/fixtures/module-unification-addon/npm/.eslintrc.js` is needed for the test `EMBER_CLI_MODULE_UNIFICATION: ember addon foo works`. 

````
        expect(actualFiles).to.deep.equal(expectedFiles);
````


Now, I think this `EMBER_CLI_MODULE_UNIFICATION: ember addon foo works` test is not needed anymore because with this PR the existing tests for Non-MU mode will also run for the MU mode.

What should I do here? Preserve both `eslintrc.js`, remove the mentioned tests or something else?

---------------------------------------------------------

I think, the end goal could be to have tests for same behaviour `MU` and `Non-MU`, so there are not existing tests that are skipped on the MU mode.

This is what I am trying to achieve with this and coming PRs. Please, correct me if I am wrong with this assumption.

Reply:
Added `.eslintrc.js` in blueprint and fixture at https://github.com/ember-cli/ember-cli/pull/8231/commits/6af2c66b5ed71dcd3d1be5f6401cbd63ec440eaf